Update sys.status.mdx (#141)
comcode-org/hackmud_wiki: docs/scripting/trust scripts/sys.status.mdx

sys.status returns the status of the user that runs it.

### Security Level

HIGHSEC

## Syntax

### CLI

```
sys.status
```

### Script

```
#fs.sys.status()
```

### Parameters

There are no known parameters.

### Return

> sys.specs returns a `String`

Reviewer: I'd write the type as follows:
```suggestion
sys.specs returns a string
```

Otherwise there might be some confusion about `String` vs. the primitive string type

## Syntax

> On the Command Line Interface:

Reviewer 2: ```suggestion
### CLI
```

```
sys.status
```

> As a subscript call:

Reviewer 2: ```suggestion
### Script
```

```
#fs.sys.status()
```

### Parameters

There are no known parameters.

### Return

> In the main terminal window, sys.status returns one of the following:

Reviewer 2: ```suggestion
sys.status returns a string in the CLI or an object in a script.

#### CLI

`-system status nominal-`

#### Script

```
{
  hardline: 0,
  tutorial: "",
  breach: false
}```

```

Reviewer 2: This may make things much worse but it's intended to bring it inline with the template

> ```

Reviewer 2: ```suggestion
## Example

```
```

Brings back example hopefully
